pkgs/tools/misc/turbo/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, fetchFromGitHub, buildGoModule, git, nodejs, protobuf, protoc-gen-go, protoc-gen-go-grpc }:
+let
+  version = "1.6.3";
```

Comment:
Please inline this as well, and use `buildGoModule rec {..`

Reply:
@dit7ya I added the the rec variable update. Regarding running on an existing project, i think you need `turbo.json` https://turbo.build/repo/docs/getting-started/add-to-project. You should get a message with "FULL TURBO" at the end:
```
❯ turbo run build --filter=next13
• Packages in scope: next13
• Running build in 1 packages
• Remote caching enabled
next13:build: cache hit, replaying output 8afe13083e6aec2b
next13:build:
next13:build: > next13@0.1.0 build /home/dane/code/trepo/apps/next13
next13:build: > next build
next13:build:
next13:build: warn  - You have enabled experimental feature (appDir) in next.config.js.
next13:build: warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.
next13:build: info  - Thank you for testing `appDir` please leave your feedback at https://nextjs.link/app-feedback
next13:build:
next13:build: info  - Creating an optimized production build...
next13:build: info  - Compiled successfully
next13:build: info  - Linting and checking validity of types...
next13:build: error - ESLint: Plugin "@next/next" was conflicted between ".eslintrc.json » eslint-config-next/core-web-vitals » plugin:@next/next/core-web-vitals" and "../../.eslintrc.js » eslint-config-custom » eslint-config-next » plugin:@next/next/recommended".
next13:build: info  - Collecting page data...
next13:build: info  - Generating static pages (0/3)
next13:build: info  - Generating static pages (3/3)
next13:build: info  - Finalizing page optimization...
next13:build:
next13:build: Route (app)                                Size     First Load JS
next13:build: ─ ○ /                                      0 B                0 B
next13:build: + First Load JS shared by all              66.5 kB
next13:build:   ├ chunks/main-app-5d10f268f7daaf39.js    64.4 kB
next13:build:   └ chunks/webpack-4c4738e139af2bb8.js     2.07 kB
next13:build:
next13:build: Route (pages)                              Size     First Load JS
next13:build: ┌ ○ /404                                   178 B          79.7 kB
next13:build: └ λ /api/hello                             0 B            79.6 kB
next13:build: + First Load JS shared by all              79.6 kB
next13:build:   ├ chunks/main-d22763246e87f499.js        77.3 kB
next13:build:   ├ chunks/pages/_app-ee2cb9d648c4ddfb.js  192 B
next13:build:   └ chunks/webpack-4c4738e139af2bb8.js     2.07 kB
next13:build:
next13:build: λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
next13:build: ○  (Static)  automatically rendered as static HTML (uses no initial props)
next13:build:

 Tasks:    1 successful, 1 total
Cached:    1 cached, 1 total
  Time:    210ms >>> FULL TURBO
```